index.rst

```diff
@@ -222,6 +233,7 @@ In different implementations these tables may exist directly, as a pass-through
 
 The point is that users/developers can write their SQL queries against this fixed schema.
 
+[KSK]  I know I'm exposing a bias here, but a ``Butler`` without ``get`` and ``put`` methods doesn't seem like a butler (it doesn't abstract I/O).  I can imagine this being a mixin for use with a ``PrimitiveButler`` or ``StorageButler``, but it seems confusing to call it a butler.
```

Comment:
Yup, I definitely think of all these "butlers" as just interfaces that could be optionally combined (sort of like mixins, but so far without any implied implementation with them).  I'd be happy to call it something else, but I think @pschella's mental picture of how all these fit together is at least a bit different from mine, so we should make sure he chimes in.